src/main/webapp/css/page.css

```diff
@@ -0,0 +1,31 @@
+:root {
+  --body-text: rgb(25, 25, 25);
+  --header-color: rgb(63, 81, 181);
+  --header-text: rgb(255, 255, 255);
+  --theme-color: rgb(255, 82, 82);
+}
+
+button {
+  background-color: var(--header-color);
+  color: var(--header-text);
+}
+
+.body {
+  margin: auto;
+  width: 75%;
+}
+
+.body-text {
+  color: var(--body-text);
+}
+
+.header {
+  color : var(--theme-color);
```

Comment:
Arrange from most specific to least specific.

Reply:
Isn't an element more specific than a class?

Reply:
Uhhh I thought it's the reverse.. We can ask Tina

Reply:
You're right! Sorry